Formula/i/idsgrep.rb

```diff
@@ -0,0 +1,37 @@
+class Idsgrep < Formula
+  desc "Grep for Extended Ideographic Description Sequences"
+  homepage "https://tsukurimashou.org/idsgrep.php.en"
+  url "https://tsukurimashou.org/files/idsgrep-0.6.tar.gz"
+  sha256 "2c07029bab12d9ceefddf447ce4213535b68d020b093a593190c2afa8a577c7c"
+  license "GPL-3.0-or-later"
```

Comment:
this should be `GPL-3.0-only`

```
/*
 * String hash table for IDSgrep
 * Copyright (C) 2012, 2013  Matthew Skala
 *
 * This program is free software: you can redistribute it and/or modify
 * it under the terms of the GNU General Public License as published by
 * the Free Software Foundation, version 3.
 *
 * This program is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 * GNU General Public License for more details.
 *
 * You should have received a copy of the GNU General Public License
 * along with this program.  If not, see <http://www.gnu.org/licenses/>.
 *
 * Matthew Skala
 * http://ansuz.sooke.bc.ca/
 * mskala@ansuz.sooke.bc.ca
 */
```